LocalSettings.php.mardi.template

```diff
@@ -1,11 +1,22 @@
 
-#Pingback
+# Pingback
 ${DOLLAR}wgWBRepoSettings['wikibasePingback'] = false;
 
-# load additional setting from LocalSettings.d
-foreach (glob("/shared/LocalSettings.d/*.php") as ${DOLLAR}filename)
+# Load additional settings from LocalSettings.d
```

Comment:
Please outline what this does concretely in comment in beginning of script: 
1. Include Localsettings 
2. Include Extensions in alphabetic order

Reply:
I just added the description and also initialized a couple of variables, to avoid errors in case no extra LocalSettings are loaded, such as in docker-importer.